tests/utils/exportCheck.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable global-require */
+/* eslint-disable no-console */
+const assert = require('assert');
+
+const utapi = require('../../');
+
+const version1Exports = {
+    UtapiServer: require('../../lib/server'),
+    UtapiClient: require('../../lib/UtapiClient'),
+    UtapiReplay: require('../../lib/UtapiReplay'),
+    UtapiReindex: require('../../lib/UtapiReindex'),
+};
+
+const version2Exports = {
+    version: 2,
+};
+
+const versionToCheck = process.argv[2];
```

Comment:
Same question as my other comment on `version` and `process.env.ENABLE_UTAPI_V2`!